bayes3d/__init__.py

```diff
@@ -1,24 +1,12 @@
 """
 .. include:: ./documentation.md
 """
-from .transforms_3d import *
+
+from .camera import *
+from .likelihood import *
 from .renderer import *
 from .rgbd import *
-from .likelihood import *
-from .camera import *
+from .transforms_3d import *
 from .viz import *
-from . import utils
-from . import distributions
-from . import scene_graph
-from . import colmap
```

Comment:
I think this is breaking many of our scripts and tutorials now because expressions like `b.distributions` will throw `AttributeError: module 'bayes3d' has no attribute 'distributions'`. Though don't worry, I'm going to add these imports back in PR  #43 :)